sound/soc/sof/topology.c

```diff
@@ -289,7 +289,7 @@ static const struct sof_dai_types sof_dais[] = {
 	{"ALH", SOF_DAI_INTEL_ALH},
 	{"SAI", SOF_DAI_IMX_SAI},
 	{"ESAI", SOF_DAI_IMX_ESAI},
-	{"ACP", SOF_DAI_AMD_BT},
+	{"ACPBT", SOF_DAI_AMD_BT},
```

Comment:
No other devices are used "ACP" till now

Reply:
Till now SOF_DAI_AMD_BT is not used, now we are going to use this dai.
For better readability changing to "ACPBT" instead of "ACP".
Other devices are not affected as this BT DAI is not used till now.

Reply:
@plbossart @ranj063 
Could you please help to review the PR